code/game/machinery/doors/airlock.dm

```diff
@@ -376,8 +379,15 @@
 			if(!C.wearing_shock_proof_gloves())
 				new /datum/hallucination/shock(C)
 				return
-	if (cyclelinkedairlock)
-		if (!shuttledocked && !emergency && !cyclelinkedairlock.shuttledocked && !cyclelinkedairlock.emergency && allowed(user))
+	if(close_others)
+		for(var/obj/machinery/door/airlock/otherlock as anything in close_others)
+			if(!shuttledocked && !emergency && !otherlock.shuttledocked && !otherlock.emergency && allowed(user))
+				if(otherlock.operating)
+					otherlock.delayed_close_requested = TRUE
+				else
+					addtimer(CALLBACK(otherlock, .proc/close), 2)
```

Comment:
2 ticks?

Reply:
im LITERALLY just doing the exact same method of closing that the current mapping helper method uses as coded by @MrStonedOne i have no idea what the timer delay is for, thats just how it currently works and i see no reason to change it

Reply:
mainly for that reason. It was designed to allow dragging something behind you without the door closing. also because its intentionally not as secure as the virology version.